File: ui-lib/src/main/java/gcm/heruijun/com/ui_lib/PrintTextView.java
package gcm.heruijun.com.ui_lib;

import android.content.Context;
import android.support.annotation.Nullable;
import android.util.AttributeSet;
import android.widget.TextView;

/**
 * Created by heruijun on 2017/12/10.
 */

public class PrintTextView extends android.support.v7.widget.AppCompatTextView {

    public PrintTextView(Context context) {
        super(context);
        init();
    }

    public PrintTextView(Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
        init();
    }

    public PrintTextView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        init();
    }

    private void init() {

    }

}

File: BaseLibrary/src/main/java/com/gcm/heruijun/base/widgets/PrintTextView.kt
package com.gcm.heruijun.base.widgets

import android.content.Context
import android.util.AttributeSet
import android.widget.TextView

/**
 * Created by heruijun on 2017/12/10.
 */

class PrintTextView : android.support.v7.widget.AppCompatTextView {

    constructor(context: Context) : super(context) {
        init()
    }

    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
        init()
    }

    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
        init()
    }

    private fun init() {

    }

}
